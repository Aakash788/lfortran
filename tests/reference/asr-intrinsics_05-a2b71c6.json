{
    "basename": "asr-intrinsics_05-a2b71c6",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/intrinsics_05.f90",
    "infile_hash": "a0b26c89376dc415c9180441ee41b111288a075487d7393f2d5e77e4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-intrinsics_05-a2b71c6.stderr",
    "stderr_hash": "3a73f12ef5c63f13adf2e3a2e6962f4fe4799ce80d2d3e9de7dd5ad1",
    "returncode": 2
}