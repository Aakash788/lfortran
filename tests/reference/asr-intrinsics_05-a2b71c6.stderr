semantic error: The kind of first argument of `ior` intrinsic must be the same as second argument
 --> tests/errors/intrinsics_05.f90:2:14
  |
2 |     print *, ior(1, 1_8)
  |              ^^^^^^^^^^^ 
